.github/workflows/autorelease.yaml

```diff
@@ -23,3 +23,9 @@ jobs:
           body: ${{ env.RELEASE_NOTES }}
           tag_name: ${{ env.RELEASE_TAG }}
           name: ${{ env.RELEASE_TAG }}
+      - name: Trigger Go proxy cache refresh
+        run: |
+          LATEST_SDK_TAG=$(curl -sSfL https://api.github.com/repos/mongodb/atlas-sdk-go/releases/latest | jq -r '.tag_name')
```

Comment:
we could fo LATEST_SDK_TAG=${{ env.RELEASE_TAG }} but I prefer in this way so the step is more "independent" and we can check both versions are equal

Reply:
Wondering if `go get` is enough, but if you have tested this method then we good.

Reply:
this was recommended by @gssbzn